R/zchunk_L131.nonco2_proc_R_S_T_Y.R

```diff
@@ -36,53 +40,155 @@ module_emissions_L131.nonco2_proc_R_S_T_Y_DISABLED <- function(command, ...) {
+      filter(complete.cases(.)) %>%  #delete rows with NAs
+      group_by(GCAM_region_ID, year, Non.CO2, agg_sector) %>%
+      summarise(EDGAR_emissions=sum(value)) %>%
+      mutate(EDGAR_emissions=EDGAR_emissions * CONV_GG_TG) -> #convert from gg to tg
+      L131.EDGAR.mlt
+
+    L131.EDGAR.mlt$GCAM_region_ID <- as.integer(L131.EDGAR.mlt$GCAM_region_ID)
+    L131.EDGAR.mlt$year <- as.integer(L131.EDGAR.mlt$year)
+
+    # Fifth: map in all data and compute emissions (EDGAR_emissions * tech_share)
+    GCAM_sector_tech %>%
+      select(supplysector, subsector, stub.technology, EDGAR_agg_sector, EPA_agg_sector, EPA_agg_fuel_ghg) %>%
+      filter(EDGAR_agg_sector %in% c("industry_processes" ,"chemicals", "landfills", "wastewater", "aerosols",
+                                     "metals", "foams", "solvents", "semiconductors")) %>%
+      repeat_add_columns(tibble::tibble(Non.CO2=c("CH4","N2O","NMVOC","NOx","SO2","CO","VOC"))) %>%
+      group_by(supplysector, subsector, stub.technology, Non.CO2) %>%
+      left_join(L131.nonco2_pct_R_prc_S_S_2005, by = c("supplysector", "subsector", "stub.technology", "Non.CO2")) %>%
+      repeat_add_columns(tibble::tibble(year= c(emissions.EDGAR_YEARS))) %>%
+      repeat_add_columns(tibble::tibble(GCAM_region_ID = c(GCAM_region_names$GCAM_region_ID))) %>%
+      group_by(GCAM_region_ID, EDGAR_agg_sector.x, Non.CO2, year) %>%
+      left_join(L131.EDGAR.mlt, by = c("GCAM_region_ID", "EDGAR_agg_sector.x" ="agg_sector", "Non.CO2", "year")) %>%
+      na.omit() %>%  #delete rows with NAs
+      mutate(input.emissions = EDGAR_emissions * tech_share) %>%
+      group_by(GCAM_region_ID, supplysector, subsector, stub.technology, Non.CO2, year) %>%
+      summarise(input.emissions=sum(input.emissions)) %>%
+
+      # Produce outputs\
+       add_title("GHG emissions by GCAM region / sector / technology / historical year") %>%
+      add_units("Tg") %>%
+      add_comments("Calculate historical emissions from the processing sector by GCAM
+                   technology,computed from EDGAR emissions data and EPA emissions factors.") %>%
       add_legacy_name("L131.nonco2_tg_R_prc_S_S_Yh") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
+      add_precursors("common/GCAM_region_names",
+                     "common/iso_GCAM_regID",
+                     "emissions/EDGAR_sector",
+                     "emissions/EPA_ghg_tech",
+                     "emissions/GCAM_sector_tech",
+                     "emissions/EDGAR_NH3",
+                     "emissions/EDGAR_CH4",
+                     "emissions/EDGAR_N2O",
+                     "emissions/EDGAR_NMVOC",
+                     "emissions/EDGAR_NOx",
+                     "emissions/EDGAR_SO2",
+                     "emissions/EDGAR_CO",
+                     "emissions/EPA_FCCC_IndProc_2005") %>%
       # typical flags, but there are others--see `constants.R`
       add_flags(FLAG_NO_TEST, FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
```

Comment:
You need to remove the `FLAG_NO_TEST`! If this is still here, it means the chunk's output wasn't getting run through the testing system.